src/screens/FeedScreen/Header.js

```diff
@@ -537,13 +537,13 @@ const styles = StyleSheet.create({
     flexDirection: 'row'
   },
   textFollow: {
-    color: colors.bluePrimary,
+    color: COLORS.blue,
     fontSize: normalizeFontSize(14),
     fontStyle: 'normal',
     fontWeight: '500'
   },
   textFollowing: {
-    color: colors.greySubtile1,
+    color: COLORS.grey9,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color values have been updated from `colors.bluePrimary` and `colors.greySubtile1` to `COLORS.blue` and `COLORS.grey9` respectively. These changes are consistent with the update in the import statement.
```diff
- color: colors.bluePrimary,
+ color: COLORS.blue,
...
- color: colors.greySubtile1,
+ color: COLORS.grey9,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->